pkgs/development/python-modules/pandas-ta/default.nix

```diff
@@ -14,16 +14,14 @@
 
 buildPythonPackage rec {
   pname = "pandas-ta";
-  version = "0.3.14";
+  version = "0.3.14b";
   pyproject = true;
 
   disabled = pythonOlder "3.7";
 
-  src = fetchFromGitHub {
-    owner = "twopirllc";
-    repo = "pandas-ta";
-    tag = version;
-    hash = "sha256-1s4/u0oN596VIJD94Tb0am3P+WGosRv9ihD+OIMdIBE=";
+  src = fetchurl {
+    url = "https://www.pandas-ta.dev/assets/zip/pandas_ta-${version}.tar.gz";
```

Comment:
Since this seems to went kinda closed source, do we need to adjust meta?

Reply:
marking it as unfree